controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -46,50 +46,59 @@ controls:
     controls:
     - id: 5.2.1
       title: Minimize the admission of privileged containers
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_privileged_containers
       level: level_1
     - id: 5.2.2
       title: Minimize the admission of containers wishing to share the host process
         ID namespace
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_process_id_namespace
       level: level_1
     - id: 5.2.3
       title: Minimize the admission of containers wishing to share the host IPC namespace
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_ipc_namespace
       level: level_1
     - id: 5.2.4
       title: Minimize the admission of containers wishing to share the host network
         namespace
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_network_namespace
       level: level_1
     - id: 5.2.5
       title: Minimize the admission of containers with allowPrivilegeEscalation
-      status: pending
+      status: partial
       rules: []
+        - scc_limit_privilege_escalation
       level: level_1
     - id: 5.2.6
       title: Minimize the admission of root containers
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        - scc_limit_root_containers
       level: level_2
     - id: 5.2.7
       title: Minimize the admission of containers with the NET_RAW capability
-      status: pending
-      rules: []
+      status: partial
```

Comment:
I think when using partial we should add a note (there's a YAML attribute in the schema for that) or alternatively did you mean manual?